aws_lambda_builders/workflows/nodejs_npm/workflow.py

```diff
@@ -23,14 +23,20 @@
     NodejsNpmrcAndLockfileCopyAction,
     NodejsNpmrcCleanUpAction,
 )
-from aws_lambda_builders.workflows.nodejs_npm.exceptions import OldNpmVersionError
 from aws_lambda_builders.workflows.nodejs_npm.npm import SubprocessNpm
 from aws_lambda_builders.workflows.nodejs_npm.utils import OSUtils
 
 LOG = logging.getLogger(__name__)
 
 # npm>=8.8.0 supports --install-links
 MINIMUM_NPM_VERSION_INSTALL_LINKS = (8, 8)
+UNSUPPORTED_NPM_VERSION_MESSAGE = (
+    "Building in source was enabled, however the "
+    "currently installed npm version does not support "
+    "--install-links. Please ensure that the npm "
+    "version is at least 8.8.0. Switching to build "
+    "in a scratch directory."
```

Comment:
Link to NPM documentation might be useful

Reply:
The closest thing to docs that would indicate the minimum version would be this: https://docs.npmjs.com/cli/v8/using-npm/changelog#v880-2022-04-27

I can add this to the message. Additionally I noticed that I've left the message as "switching to scratch directory", which isn't 100% true for Node.js. I can change this to "switching to building outside of the source directory".